Code/GraphMol/FMCS/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@ rdkit_library(FMCS
               FMCS.cpp Seed.cpp MaximumCommonSubgraph.cpp SubstructMatchCustom.cpp
               LINK_LIBRARIES Depictor FileParsers ChemTransforms SubstructMatch)
 
-rdkit_headers(FMCS.h
+rdkit_headers(DebugTrace.h
```

Comment:
While making this change, we might as well get rid of the headers that aren't needed in client code.
I think the only two headers needed here are: FMCS.h and Graph.h

Reply:
That makes sense. In fact, I thought the same, but as I wasn't sure why the other headers were installed in the first place, I thought it would be safer to add the missing ones.